contacts/tasks.py

```diff
@@ -0,0 +1,17 @@
+from django.core.mail import EmailMessage
+from phone_book.celery import app
+from celery.utils.log import get_task_logger
+
+logger = get_task_logger(__name__)
+
+
+@app.task(name='send')
+def send(mail_subject, to_email, message):
+
+    email = EmailMessage(
+        mail_subject, message, to=[to_email]
+    )
+    email.send()
+    return "Email Sent"
```

Comment:
What is the purpose of this return statement?

Reply:
when i run the worker it shows me this return message not necessary but i want a custom return message for check that's why but removing now

Reply:
![image](https://user-images.githubusercontent.com/41616030/55553258-2a6ad280-56f9-11e9-93c4-c2e1586d7d8f.png)

Reply:
Okay, you can use the return statement. You can improve it by returning and displaying a more descriptive message.

Reply:
`    return "Email Has been sent from Task of sending email
`
sure